spec/fixtures/alma/current_fixtures.json

```diff
@@ -16452,5 +16452,695 @@
+      ],
+      "author_s":[
+        "Ptolemy, active 2nd century",
+        "Pirckheimer, Willibald, 1470-1530",
+        "Regiomontanus, Joannes, 1436-1476",
+        "Martini, Giuseppe, 1870-1944",
+        "Kane, Grenville, 1854-1943"
+      ],
+      "marc_relator_display":[
+        "Author"
+      ],
+      "uniform_title_s":[
+        "Geographia. Latin"
+      ],
+      "title_display":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo / Bilibaldo Pirckeymhero interprete ; annotationes Ioannis de Regio Monte in errores commissos a Iacobo Angelo in translatione sua."
+      ],
+      "title_a_index":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo"
+      ],
+      "title_sort":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo / Bilibaldo Pirckeymhero interprete ; annotationes Ioannis de Regio Monte in errores commissos a Iacobo Angelo in translatione sua."
+      ],
+      "title_no_h_index":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo / Bilibaldo Pirckeymhero interprete ; annotationes Ioannis de Regio Monte in errores commissos a Iacobo Angelo in translatione sua."
+      ],
+      "title_t":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo / Bilibaldo Pirckeymhero interprete ; annotationes Ioannis de Regio Monte in errores commissos a Iacobo Angelo in translatione sua."
+      ],
+      "title_citation_display":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo"
+      ],
+      "compiled_created_t":[
+        "Claudii Ptolemaei Geographicae enarrationis libri octo / Bilibaldo Pirckeymhero interprete ; annotationes Ioannis de Regio Monte in errores commissos a Iacobo Angelo in translatione sua."
+      ],
+      "pub_created_display":[
+        "Argentoragi [Strasbourg] : Iohannes Grieningerus, communibus Iohannis Koberger impensis excudebat, 1525."
+      ],
+      "pub_created_s":[
+        "Argentoragi [Strasbourg] : Iohannes Grieningerus, communibus Iohannis Koberger impensis excudebat, 1525."
+      ],
+      "pub_citation_display":[
+        "Argentoragi [Strasbourg]: Iohannes Grieningerus, communibus Iohannis Koberger impensis excudebat"
+      ],
+      "pub_date_display":[
+        "1525"
+      ],
+      "pub_date_start_sort":[
+        "1525"
+      ],
+      "cataloged_tdt":[
+        "2021-07-12T21:27:48Z"
+      ],
+      "format":[
+        "Book"
+      ],
+      "electronic_access_1display":[
+        "{\"http://arks.princeton.edu/ark:/88435/np193c809\":[\"arks.princeton.edu\",\"EXKA Ptolemy 1525\"],\"http://arks.princeton.edu/ark:/88435/kw52jc00n\":[\"arks.princeton.edu\",\"EXKA Ptolemy 1525a\"]}"
+      ],
+      "electronic_access_index":[
+        "http://arks.princeton.edu/ark:/88435/np193c809 EXKA Ptolemy 1525",
+        "http://arks.princeton.edu/ark:/88435/kw52jc00n EXKA Ptolemy 1525a"
+      ],
```

Comment:
@tpendragon I'm trying to replicate the fix you did in this pr. In solr electronic_access_1_display has more information indexed while you are adding only what is in 856. There is no electronic_access_index field. Why did you add it?

Reply:
ok I see the key `iiif_manifest_paths` needs to be removed from the electronic_access_1display.